api/environments/identities/models.py

```diff
@@ -208,4 +208,10 @@ def update_traits(self, trait_data_items):
     def send_to_dynamodb(self):
         if dynamo_identity_table:
             identity_dict = build_identity_dict(self)
+            # Add the pk for dynamodb
+            identity_dict.update(
+                {
+                    "pk": f"{identity_dict['environment_api_key']}_{identity_dict['identifier']}"
```

Comment:
Could we move this to a static method on the IdentityModel class?

Reply:
@matthewelwell  I think property makes more sense?